website/docs/foundations/colors/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,94 @@
+## Color styles and tokens
+
+Color naming syntax and use can vary depending on context and tooling. This overview will clarify what naming conventions are typically used and how they align to the HDS standards implemented in our libraries.
+
+In Figma, reusable colors are referred to as “Styles.” They are categorized and stored in the right sidebar for fills, borders, and other properties that accept color values. Styles directly reference a HEX value but cannot reference each other (known as aliasing for Tokens). This means when we provide Figma Styles for semantic usage (like `Foreground/Primary`), the HEX value (`#3B3D45`) is directly referenced instead of the Core Palette color (`Palette/Neutral/Neutral 600`). 
+
+In code, we refer to styles as “Tokens.” They are generated from agnostic files, in our case JSON, and create atomic colors regardless of what platform or framework is used. For example, Core Palette colors like `--token-color-palette-neutral-600` are directly referenced in `--token-color-foreground-primary`. 
```

Comment:
Suggestion:
For clarity, perhaps change:
"They are generated from agnostic files"
to
"They are generated from platform-agnostic files"

(I know you explain it after but I was initially confused until I got to that part of the sentence as "agnostic" is normally qualified as to what something is agnostic to.)

Reply:
Fair point, ill update :)